tests/mvc/collection/collection.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, bender, sinon */
+
+'use strict';
+
+var modules = bender.amd.require( 'mvc/collection', 'mvc/model' );
+
+describe( 'add', function() {
+	it( 'should change length and enable get', function() {
+		var Model = modules[ 'mvc/model' ];
+
+		var box = getCollection();
+
+		expect( box ).to.have.length( 0 );
```

Comment:
I've replaced all these `expect( box.length ).to.equals( 0 );` assertions with more explicit ones.